src/main/webapp/index.html

```diff
@@ -6,13 +6,13 @@
         <link rel="stylesheet" href="style.css">
             <script src="script.js"></script>
     </head>
-    <body >
+    <body>
         <div class="content">
         <h1>PromoTube</h1>
-        <form id="searchForm">
-            <input type="text" id="formInput" name="formInput" value="Enter Channel URL">
-            <input type="submit" value="Enter">
-        </form>
+        <div id="searchForm">
```

Comment:
so there's no <form> element now?  how can you have <input> elements if not enclosed in an <form>

Reply:
You don't need them to be in a form. When a form is submitted the page refreshes and it causes problems.

Reply:
what issue did you run into?

Reply:
When the input is in a form the submit button causes the page to refresh which makes it hard to display the codes if they immediately disappear.

Reply:
OK.  Mind adding a comment saying that's why you chose not to put it in a `<form>`